tests/door43_tools_tests/test_templaters.py

```diff
@@ -48,9 +48,9 @@ def testTemplaterTaComplete(self):
         test_file_path = self.extractZipFiles(test_folder_name)
         success = self.doTemplater('ta', test_file_path)
         self.verifyTaTemplater(success, expect_success, self.out_dir,
-                               ['checking.html', 'intro.html', 'process.html', 'translate.html'])
+                               ['01-intro.html', '02-process.html', '03-translate.html', '04-checking.html'])
```

Comment:
So basically this test wasn't about making sure the files were prefixed with numbers, but a test that has been there before I made this change, but the file names needed to be updated for this test. In any regard, I added a new test below this one to verify the order of the manuals in the TOC.